js/transform.js

```diff
@@ -222,6 +232,15 @@ function displayForm(e) {
 }
 
 function generateAndDisplayForm(formId, dest) {
+    const INTEGER_AFTER_UNDERSCORE_REGEX = /(.*)_([0-9]+)/;
```

Comment:
[Pebble] Generally, a constant that is not changing in the function scope should be declared outside of that scope so that it's easier to understand what is dynamic as you read through the function. I recommend that unless there's a specific reason here!